python/lsst/ts/dream/csc/mock/mock_dream.py

```diff
@@ -179,14 +183,14 @@ async def ready_for_data(self, ready: bool):
         """
         self.log.info(f"readyForData called with param ready {ready}")
 
-    async def data_archived(self):
+    async def data_archived(self) -> None:
```

Comment:
This got renamed no?

Reply:
Yes, I was waiting for you to finish your review of `ts_dream_common`. Done now.